src/css/yeldaWebchatInjector.css

```diff
@@ -1,5 +1,5 @@
-@import url(https://use.fontawesome.com/releases/v5.3.1/css/all.css);
-
+@import url('https://yelda-chat.s3.eu-west-3.amazonaws.com/fontawesome-free-5.3.1-web/css/all.min.css');
+@import 'lightgallery.js/dist/css/lightgallery.min.css';
```

Comment:
Sorry for the delay in the reply

> css version is just the copy of css file from src folder, it does not include the imported css of fontawesome, minified version is the css which include all the imported css

This is not normal : the minified version of a script or a css is that version ... minified, it should contain the same content, but minified...

So the `../dist/css/yeldaWebchatInjector.css` moved in dist should not be `the copy of css file from src folder` but the result of the css **build**

And `../dist/css/yeldaWebchatInjector.min.css` that same file minified

in other word, when i run `npm run build`, the js and css version should be perfectly usable in html

see test/index.html and open it in your browser, you'll see that it works well 
Now edit "../dist/css/yeldaWebchatInjector.css" to "../dist/css/yeldaWebchatInjector.min.css"/ and "../dist/js/yeldaWebchatInjector.min.js" to "../dist/js/yeldaWebchatInjector.js" => it should work exactly the same way (excepting that the min version are bit smaller because they have been minified.

in our case, it does not works perfectly, because `lightgallery.min.css` is not found

![image](https://user-images.githubusercontent.com/5340081/164000470-3290ae1d-7d21-44cd-a0b0-17eeef6cc342.png)

So in short, for this PR to be valid, we need to the /dist css to be the **build** of the css file, not the copy of the source.

If it's not clear, please ping me !

--
 
Another important point : the objective here is to reduce the size of our CSS for the clients who complied about it.
So instead of adding lightgallery css to our CSS, let's load it from it's official CDN, see https://cdnjs.com/libraries/lightgallery/1.2.0 => 

```css
/* before */
@import url('https://yelda-chat.s3.eu-west-3.amazonaws.com/fontawesome-free-5.3.1-web/css/all.min.css');
@import 'lightgallery.js/dist/css/lightgallery.min.css';

/* after : let's load it from it's cdn */
@import url('https://yelda-chat.s3.eu-west-3.amazonaws.com/fontawesome-free-5.3.1-web/css/all.min.css');
@import url('https://cdnjs.cloudflare.com/ajax/libs/lightgallery/1.2.0/css/lightgallery.min.css');
```